services/QuillCMS/app/controllers/responses_controller.rb

```diff
@@ -75,7 +75,7 @@ def responses_for_question
   def multiple_choice_options
     multiple_choice_options = Rails.cache.fetch("questions/#{params[:question_uid]}/multiple_choice_options", :expires_in => 900) do
       optimal_responses = Response.where(question_uid: params[:question_uid], optimal: true).order('count DESC').limit(2).to_a
-      sub_optimal_responses = Response.where(question_uid: params[:question_uid], optimal: false).order('count DESC').limit(2).to_a
+      sub_optimal_responses = Response.where(question_uid: params[:question_uid], optimal: [false, nil]).order('count DESC').limit(2).to_a
```

Comment:
This sort of feels like it might be a back-end data issue.  Is there a symantic distinction between `optional: false` and `optimal: nil`, or are we just inconsistent about which one we use?  If the former, then this probably makes sense, but if the latter, I'd prefer to correct/standardize the data rather than paper over it.

Reply:
There is a semantic difference. Only human-graded responses can be `optimal: false` - the algorithms don't know whether a response is right or wrong. For the most part, algorithmically-graded answers are suboptimal, but we do have some instances in the diagnostic where focus points and incorrect sequences are used to find to only apply correct concept results.

Reply:
The issue that this PR is fixing is that there aren't always human-graded suboptimal answers, to further clarify.

Reply:
That makes sense.  In the longer-term, like once we have Comprehension, the assumption that algorithm-graded answers must always be sub-optimal may not hold true, but it's certainly true for now, so this seems like a good approach.